src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html

```diff
@@ -10,15 +10,38 @@ <h3 mat-dialog-title>{{data.description.title}}</h3>
       </mat-select>
     </mat-form-field>
   </div>
+  <div>
+    <mat-form-field class="example-full-width">
+      <mat-label>Предмет</mat-label>
+      <mat-select [(ngModel)]="data.data.subject_id" name="subject_id" #subject="ngModel">
+        <mat-option *ngFor="let subject of subjects" [value]="subject.subject_id">
+          {{subject.subject_name}}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+  </div>
   <div>
     <mat-form-field>
-      <input matInput [matDatepicker]="picker" placeholder="Виберіть дату" disabled>
+      <input matInput [matDatepicker]="picker"
+             placeholder="Виберіть дату"
+             name="start_date" ngModel
+             #start_date="ngModel"
+             #dateCtrl="ngModel"
+             required
+             [(ngModel)]="data.data.start_date">
       <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
       <mat-datepicker #picker disabled="false"></mat-datepicker>
+      <mat-error *ngIf="dateCtrl.errors?.required && dateCtrl.touched">Виберіть дату!</mat-error>
     </mat-form-field>
   </div>
-  <div class="ngx-timepicker-field-example">
-    <ngx-timepicker-field [format]="24"></ngx-timepicker-field>
+  <div>
+    <mat-form-field>
+      <input matInput placeholder="Виберіть годину"
+             name="start_time" ngModel
+             #start_time="ngModel"
```

Comment:
Looks like start_time ref is not used as well.

Reply:
The same solution as in comment above